### PR TITLE
Clean test base

### PR DESCRIPTION
**Pull request by kornys**
### Type of change

- Refactoring

### Description

Remove or move methods from testBase class and keep only help and assert methods there. It is more readable from my POV now

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by kornys** [comment]
@enmasse-ci run tests profile=shared test_only

File: systemtests/src/main/java/io/enmasse/systemtest/utils/AddressSpaceUtils.java [hunk 1]
```diff
@@ -321,4 +322,53 @@ public static String getConnectorStatuses(AddressSpace addressSpace) {
...
+            messagingEndpoint = Kubernetes.getInstance().getExternalEndpoint(externalEndpointName);
+        }
+        if (TestUtils.resolvable(messagingEndpoint)) {
+            return messagingEndpoint;
+        } else {
+            return Kubernetes.getInstance().getEndpoint("messaging-" + AddressSpaceUtils.getAddressSpaceInfraUuid(addressSpace), addressSpace.getMetadata().getNamespace(), "amqps");
+        }
+    }
+
+    public static String getOCConsoleRoute() {
```
Comment (famartinrh) [inline systemtests/src/main/java/io/enmasse/systemtest/utils/AddressSpaceUtils.java]: this doesn't belong here, it fits better in kubernetes or openshift class or some other utils class

Comment (kornys) [reply systemtests/src/main/java/io/enmasse/systemtest/utils/AddressSpaceUtils.java]: yup you are right

**Comment by kornys** [comment]
@enmasse-ci run tests profile=isolated

**Comment by kornys** [comment]
@enmasse-ci run tests profile=isolated test_only

**Comment by kornys** [comment]
@enmasse-ci run tests testcase=CustomResourceDefinitionAddressesTest profile=isolated test_only

**Pull request merged**